doc/newsfragments/dict_match_for_ignored_fields.rst

```diff
@@ -0,0 +1 @@
+* When comparing 2 dictionaries even some fields are marked as "ignored", the corresponding values should also be compared, thus the result of `DictMatch` can be rendered on Web UI clearly that comparable items are presented side by side.
```

Comment:
We don't need the asterisk in news fragment. And the description here is a bit too implementation oriented. Which assertion are we fixing here (dict.match?), what was the original issue (nested value in ignored?), try to describe the change from user perspective please.

Reply:
ammended